### PR TITLE
Removed/replaced remaining instances of printStackTrace

### PR DESCRIPTION
**Pull request by andrewbp**

### CODE REVIEW

File: src/main/java/emissary/util/search/FastBoyerMoore.java [hunk 1]
```diff
@@ -289,13 +291,13 @@ public static void main2(final String[] args) {
                     scanner.scan(dataBytes, pos, hits);
                     for (int i = 0; i < hits.size(); i++) {
                         final int[] tmp = hits.get(i);
-                        System.out.println("Hit At:" + tmp[0] + " id: " + tmp[1]);
+                        logger.info("Hit At: {} id: {}", tmp[0], tmp[1]);
                     }
                     hits.clear();
                 }
             } catch (Exception e) {
                 System.out.println("Exception in test:" + e);
```
Comment (jpdahlke) [inline src/main/java/emissary/util/search/FastBoyerMoore.java:299]: Should this line be removed as well?

File: src/main/java/emissary/command/RunCommand.java [hunk 2]
```diff
@@ -54,7 +54,7 @@ public void run(JCommander jc) {
         } catch (ClassNotFoundException e) {
             throw new RuntimeException("Could not find fully qualified class named " + clazzName);
         } catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException | NoSuchMethodException | SecurityException e) {
-            e.printStackTrace();
+            LOG.error("Problem calling main from " + clazzName + " with " + Arrays.toString(clazzArgs), e);
```
Comment (jpdahlke) [inline src/main/java/emissary/command/RunCommand.java:57]: In situations like these, a pattern we've used before is to build a string once and then send that string to both the log and pass it to the exception.

File: src/main/java/emissary/util/search/FastBoyerMoore.java (hunk 1 above)
Comment (andrewbp) [reply src/main/java/emissary/util/search/FastBoyerMoore.java:299]: Whoops, yes. Is the goal to remove all console logging?

**Comment by andrewbp** [comment]
Minor changes, more efficient strings.

File: src/main/java/emissary/server/EmissaryServer.java [hunk 3]
```diff
@@ -161,7 +161,7 @@ public Server startServer() {
             server.setStopAtShutdown(true);
             server.setStopTimeout(10000l);
             if (this.cmd.shouldDumpJettyBeans()) {
-                server.dump(System.out);
+                LOG.debug(server.dump());
```
Comment (dev-mlb) [inline src/main/java/emissary/server/EmissaryServer.java:164]: I think this should be info

**Review by dev-mlb — CHANGES_REQUESTED** [review]

File: src/test/java/emissary/test/core/UnitTest.java [hunk 4]
```diff
@@ -147,7 +147,7 @@ public synchronized void setupSystemProperties() {
         try {
             ServerCommand.parse(ServerCommand.class, "-m", "cluster").setupCommand();
         } catch (InstantiationException | IllegalAccessException | ClassNotFoundException e) {
-            e.printStackTrace();
+            logger.error("Unable to setup Emissary environment", e);
             fail("Unable to setup Emissary environment");
```
Comment (dev-mlb) [inline src/test/java/emissary/test/core/UnitTest.java:151]:
```suggestion
            fail("Unable to setup Emissary environment", e);
```

File: src/test/java/emissary/test/core/UnitTest.java [hunk 5]
```diff
@@ -147,7 +147,7 @@ public synchronized void setupSystemProperties() {
         try {
             ServerCommand.parse(ServerCommand.class, "-m", "cluster").setupCommand();
         } catch (InstantiationException | IllegalAccessException | ClassNotFoundException e) {
-            e.printStackTrace();
+            logger.error("Unable to setup Emissary environment", e);
```
Comment (dev-mlb) [inline src/test/java/emissary/test/core/UnitTest.java:150]:
```suggestion
```

File: src/test/java/emissary/test/core/UnitTest.java (hunk 4 above)
Comment (dev-mlb) [inline src/test/java/emissary/test/core/UnitTest.java:151]:
```suggestion
            fail("Unable to setup Emissary environment", e);
```

**Review by dev-mlb — APPROVED** [review]

**Review by jpdahlke — APPROVED** [review]

**Pull request merged**